drivers/clock_control/Kconfig.stm32g0

```diff
@@ -0,0 +1,51 @@
+# Kconfig - STM32G0 PLL configuration options
```

Comment:
Look very similar to `Kconfig.stm32l4_wb`. Is it possible to use `Kconfig.stm32l4_wb` instead of `Kconfig.stm32g0`?

Reply:
yes, could be with a "Kconfig.stm32l4_wb_g0"

Reply:
For consistency, I'd prefer we keep aligned Kconfig.stm32xx and clock_xx.c files.
So either we keep them split, either we merge both.